File: EDD/DLL/src/main/java/ico/fes/modelo/Empleado.java

package ico.fes.modelo;

public class Empleado {
    
    private int id;
    private String nombre;
    private String paterno;
    private String materno;
    private float sueldoBase;
    private int annioIngreso;
    private int horasExtra;

    public Empleado() {
    }

    public Empleado(int id, String nombre, String paterno, String materno, float sueldoBase, int annioIngreso, int horasExtra) {
        this.id = id;
        this.nombre = nombre;
        this.paterno = paterno;
        this.materno = materno;
        this.sueldoBase = sueldoBase;
        this.annioIngreso = annioIngreso;
        this.horasExtra = horasExtra;
    }

    public int getAnnioIngreso() {
        return annioIngreso;
    }

    public void setAnnioIngreso(int annioIngreso) {
        this.annioIngreso = annioIngreso;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getPaterno() {
        return paterno;
    }

    public void setPaterno(String paterno) {
        this.paterno = paterno;
    }

    public String getMaterno() {
        return materno;
    }

    public void setMaterno(String materno) {
        this.materno = materno;
    }

    public float getSueldoBase() {
        return sueldoBase;
    }

    public void setSueldoBase(float sueldoBase) {
        this.sueldoBase = sueldoBase;
    }

    public int getHorasExtra() {
        return horasExtra;
    }

    public void setHorasExtra(int horasExtra) {
        this.horasExtra = horasExtra;
    }

    @Override
    public String toString() {
        return "Empleado:" + "\n\tId: " + id + "\n\tNombre: " + nombre + "\n\tPaterno: " + paterno + "\n\tMaterno: " + materno + "\n\tSueldo base: " + sueldoBase + "\n\tAño de ingreso: " + annioIngreso + "\n\tHoras extra: " + horasExtra + "\n-----------------------\n";
    }
    
    
    
}

File: EDD/Tarea3/Empleado.py
class Empleado:
    def __init__(self, numTrabajador, nombre, paterno, materno, hrsExtra, sueldo, annioIngreso):
        self.__numTrabajador = numTrabajador
        self.__nombre = nombre
        self.__paterno = paterno
        self.__materno = materno
        self.__hrsExtra = hrsExtra
        self.__sueldo = sueldo
        self.__annioIngreso = annioIngreso

    def getNumTrabajador(self):
        return self.__numTrabajador
    def setNumTrabajador(self, numTrabajador):
        self.__numTrabajador = numTrabajador

    def getNombre(self):
        return self.__nombre
    def setNumTrabajador(self, nombre):
        self.__nombre = nombre

    def getPaterno(self):
        return self.__paterno
    def setPaterno(self, paterno):
        self.__paterno = paterno

    def getMaterno(self):
        return self.__materno
    def setMaterno(self, materno):
        self.__materno = materno

    def getHrsExtra(self):
        return self.__hrsExtra
    def setNumTrabajador(self, hrsExtra):
        self.__hrsExtra = hrsExtra

    def getSueldo(self):
        return self.__sueldo
    def setNumTrabajador(self, sueldo):
        self.__sueldo = sueldo

    def getAnnioIngreso(self):
        return self.__annioIngreso
    def setNumTrabajador(self, annioIngreso):
        self.__annioIngreso = annioIngreso

    def toString(self):
        return "Empleado: [Numero: {}, Nombre: {}, Paterno: {}, Materno: {}, HrsExtra: {}, Sueldo: {}, Año Ingreso: {}]".format(
            self.__numTrabajador,
            self.__nombre,
            self.__paterno,
            self.__materno,
            self.__hrsExtra,
            self.__sueldo,
            self.__annioIngreso
        )